src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/CodeflowExtensions.cs

```diff
@@ -17,86 +17,30 @@
 #nullable enable
 namespace Microsoft.DotNet.DarcLib.VirtualMonoRepo;
 
-public static class VmrRegistrations
```

Comment:
@premun this broke the ChangeValidation validation job in the VMR: https://github.com/dotnet/dotnet/blob/edc7f50f8305d2ed059f23b7a94851d4b9301fd2/eng/tools/ChangeValidation/Validation.cs#L115

> D:\a\1\s\eng\tools\ChangeValidation\Validation.cs(115,9): error CS0103: The name 'VmrRegistrations' does not exist in the current context [D:\a\1\s\eng\tools\ChangeValidation\ChangeValidation.csproj]

We are fixing the tools in https://github.com/dotnet/dotnet/pull/3591. Can you please help? Thanks